packages/rum/src/domain/record/serialization/serializeAttribute.spec.ts

```diff
@@ -143,4 +143,13 @@ describe('serializeAttribute', () => {
       )
     })
   })
+
+  describe('iframe srcdoc masking', () => {
+    it('should mask the srcdoc when privacy override set to mask', () => {
+      const node = document.createElement('iframe')
+      node.srcdoc = '<html><body>data-foo</body></html>">'
+      node.setAttribute(PRIVACY_ATTR_NAME, 'mask')
+      expect(serializeAttribute(node, NodePrivacyLevel.MASK, 'srcdoc', DEFAULT_CONFIGURATION)).toBe('***')
+    })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** I am wondering if replacing the content of the iframe could‏ disturb the layout of the page during replay.
is it possible?

Reply:
LGTM!